.github/workflows/test.yml

```diff
@@ -52,3 +52,38 @@ jobs:
                 run: cd ./casbin && go test ./... -v -race
             -   name: Test fibernewrelic Middleware
                 run: cd ./fibernewrelic && go test ./... -v -race
+    Tests2:
```

Comment:
instead of copying the whole thing, can't you just add more lines with an additional IF condition where you ignore the other run for 1.15?

https://github.com/gofiber/storage/blob/main/.github/workflows/test.yml#L52

https://docs.github.com/en/actions/learn-github-actions/expressions#startswith

`if: startsWith(matrix.go-version, '1.15') === false`

Reply:
Thank you for feedback, I updated the tests